systers_portal/templates/account/logout.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+{% load url from future %}
+{% load i18n %}
+{% block head_title %}{% trans "Sign Out" %}{% endblock %}
+{% block content %}
+  <div class="mt40"></div>
```

Comment:
To comply with the rest of the account templates, please remove the spacing here.